resfulservice/src/graphql/resolver/dataset/input.graphql

```diff
@@ -13,7 +13,13 @@ input filesetQueryInput {
 
 input datasetStatusInput {
   "Dataset workflow status"
-  status: STATUS
+  status: STATUS,
+  "Filesets page to return"
+  pageNumber: Int
+  "Number of filesets to return per page"
+  pageSize: Int
```

Comment:
I noticed that when I leave `pageNumber` and `pageSize` out of the inputs, I get error messages, but they aren't marked as required on the query
If I leave out pageNumber: 
```
"errors": [
    {
      "message": "Cannot read properties of undefined (reading 'pageNumber')",
      "status": 500
    }
  ],
  ```
  If I leave out pageSize: 
  ```
  "errors": [
    {
      "message": "Skip value must be non-negative, but received: -9223372036854775808",
      "status": 500
    }
  ],
  ```

Reply:
Ah, I see what the issue is, let me push the changes